engine/crates/engine-v2/schema/src/conversion.rs

```diff
@@ -68,6 +68,10 @@ impl From<Config> for Schema {
 
             for key in object.keys {
                 let subgraph_id = key.subgraph_id.into();
+                // Some SDL are generated with empty keys, they're useless to us.
+                if key.fields.is_empty() {
+                    continue;
+                }
```

Comment:
Why is stuff generated with empty keys?  That kinda sounds like a bug, though maybe there's something I'm missing

Reply:
Ours doesn't, but theirs does: https://github.com/the-guild-org/gateways-benchmark/blob/main/federation-v1/gateways/apollo-router/supergraph.graphql#L59

Reply:
Ah, annoying. But ok